docs/excel/custom-functions-json.md

```diff
@@ -133,6 +133,9 @@ The `parameters` property is an array of parameter objects. The following table
 |  `dimensionality`  |  string  |  No  |  Must be either **scalar** (a non-array value) or **matrix** (a 2-dimensional array).  |
 |  `name`  |  string  |  Yes  |  The name of the parameter. This name is displayed in Excel's intelliSense.  |
 |  `type`  |  string  |  No  |  The data type of the parameter. Can be **boolean**, **number**, **string**, or **any**, which allows you to use of any of the previous three types. If this property is not specified, the data type defaults to **any**. |
+|  `optional`  | string | No | If **true**, parameter will be labelled optional. |
+
+[!NOTE] Any as the default value for `type` and optional parameters are currently available, but you may experience linting errors and not see optional parameters in Excel's formula UI when calling a function. This is projected to change in December of 2018.
```

Comment:
It's not clear to me what this note is trying to say, as currently written.  Is it intended to convey something like this? -> 

If the `type` property of an optional parameter is either not specified or is specified and set to **any**, you may notice issues such as linting errors in your IDE and optional parameters not being displayed when the function is being entered into a cell in Excel. This is projected to change in December 2018.

Reply:
Yes, trying to sort of say what you have. This is kind of a weird situation where the schema (which I think provides the intelliSense) is not currently up to date but the feature is already supported. So it appears as an error when you try to use it, even though it works.